### ISSUE TITLE
Ammunition cookoff causes dedicated server crash in kp_liberation map utilising RHS assets.

### ISSUE THREAD

**Issue by TheFloatingCheese** [acemod/ACE3#6022]
**Arma 3 Version**: `1.80` (stable)
**CBA Version:** `3.5.0.171204` (stable)
**ACE3 Version:** `3.12.0` (stable)

**Mods:**
```
- CBA_A3
- ace
- All RHS mods including all ACE compatibility patches.
- JSRS Soundmod including all RHS patches.
- Enhanced Movement
- FA-18 Super Hornet
- DonkeyPunch.INFO OpenChernarus Project
- ACEX
- Project OPFOR
- USS Nimitz
- CUP Terrains 
- CUP Maps
- USS Nimitz
- Ares Mod Achilles Expansion
- FIR AWS
- FIR Pilot & Crew Pack
- Su-35 Flanker E

```

**Description:**
- When a vehicle is destroyed completely, belonging to a convoy spawned by beginning the convoy secondary objective on the kp_liberation Chernarus map, with ammunition cookoff enabled, the server slows to a crawl and crashes out completely, requiring a restart. In game the ammunition cooks off in waves of a lot of sounds at once, and the cooked off projectiles stay floating in the air. Many of the mods in the mod list were not being used directly at the time (although they were loaded).

**Steps to reproduce:**
- Begin a game on the KP_Liberation Chernarus map with the default RHS arsenal presets, on a dedicated server, with ACE ammunition cookoff enabled.
-  Gather the required intel to start a Convoy secondary objective.
- Move to that objective and completely destroy one of the convoy vehicles such that it entirely blows up.

**Where did the issue occur?**
- Dedicated 

**Placed Modules:**
- None

**RPT log file:**
- https://gist.github.com/TheFloatingCheese/dbda0def16a9e38134f48cf0fa71af14.js
- No ACE debug to clipboard yet, will try to acquire.

**Comment by commy2** [comment]
And how do we know this is a problem with ace?

**Comment by TheFloatingCheese** [comment]
I was also thinking that. I was advised to post it here regardless. I will try and get an ace debug to clipboard if you think that would show if it was ace?

**Comment by commy2** [comment]
No. But this is a very specific way to reproduce this issue. It would help if you could simplify it by removing superfluous addons and a simple mission.

**Comment by TheFloatingCheese** [comment]
Right, will get that done this evening.

**Comment by TheFloatingCheese** [comment]
I was unable to reproduce the issue in any other scenario. It would seem it is to do with the mission and not ACE itself, as I was also unable to reproduce the issue destroying said vehicles outside of that "convoy" scenario within the mission.

**Comment by SilentSpike** [comment]
I'd be curious to know if the mission changed the `ace_cookoff_ammoCookoffDuration` setting value

**Comment by SilentSpike** [comment]
I've seen the floating rounds thing before, I think it has to do with certain ammunition types

**Comment by TheFloatingCheese** [comment]
We actually had that value set globally, first to default and then I adjusted it to 0.1 to try and fix the problem. It didn't seem to have any effect

**Comment by commy2** [comment]
>It would seem it is to do with the mission and not ACE itself?

Maybe, but I don't know.

**Comment by ace3mod** [comment]
Hello @TheFloatingCheese! There has been no activity on this ticket for over a period of 30 days. I am automatically replying to let you know we will close this ticket within 1 week due to inactivity and consider this resolved.

If you believe this is in error, please reply with the requested information.

Thank you. :robot:
